packages/plugin-backgroundcolor/src/BackgroundColor.ts

```diff
@@ -1,36 +1,16 @@
-import { JWPlugin, JWPluginConfig } from '../../core/src/JWPlugin';
-import { CommandParams } from '../../core/src/Dispatcher';
-import { VNode } from '../../core/src/VNodes/VNode';
+import { Color, ColorConfig } from '../../plugin-color/src/Color';
 import { Loadables } from '../../core/src/JWEditor';
 import { Keymap } from '../../plugin-keymap/src/Keymap';
-import { Inline } from '../../plugin-inline/src/Inline';
-import { InlineNode } from '../../plugin-inline/src/InlineNode';
-import { setStyle, getStyle, removeStyle } from '../../utils/src/utils';
-import { VRange } from '../../core/src/VRange';
-import { Format } from '../../plugin-inline/src/Format';
-import { FragmentNode } from '../../core/src/VNodes/FragmentNode';
 
-export interface ColorParams extends CommandParams {
-    color?: string; // css-valid color name
-}
-
-interface BackgroundColorConfig extends JWPluginConfig {
-    defaultBackgroundColor?: string;
-}
-
-const defaultConfiguration = {
-    defaultBackgroundColor: 'white',
-};
-
-export class BackgroundColor<T extends BackgroundColorConfig = BackgroundColorConfig> extends JWPlugin<T> {
-    static dependencies = [Inline];
-    configuration = { ...defaultConfiguration, ...this.configuration };
+export class BackgroundColor<T extends ColorConfig = ColorConfig> extends Color<T> {
+    styleName = 'background-color';
+    configuration = { defaultColor: 'white', ...this.configuration };
     commands = {
         colorBackground: {
-            handler: this.colorBackground,
+            handler: this.color,
```

Comment:
Here is why the command names are not derived from function names, thank you for demonstrating that.

Reply:
Hehe :-)